yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -427,14 +446,35 @@ func (yb *TargetYugabyteDB) InitEventChannelsMetaInfo(migrationUUID uuid.UUID, n
 			return false, nil
 		}
 
+		rowsStmt = fmt.Sprintf(
+			"SELECT count(*) FROM %s where migration_uuid='%s'", EVENTS_PER_TABLE_METADATA_TABLE_NAME, migrationUUID)
+		err = conn.QueryRow(context.Background(), rowsStmt).Scan(&rowCount)
+		if err != nil {
+			return false, fmt.Errorf("error executing stmt - %v: %w", rowsStmt, err)
+		}
+		if rowCount > 0 {
+			log.Info("table wise event meta info already created. Skipping init.")
+			return false, nil
```

Comment:
Same as above. Imagine someone adding more initialisation steps towards the end of this function.